cli/command/stack/deploy.go

```diff
@@ -611,6 +620,10 @@ func convertService(
 
 	return serviceSpec, nil
 }
+func sortStrings(strs []string) []string {
+	sort.Strings(strs)
+	return strs
+}
```

Comment:
I don't really like this function. It hides the side effect from `sort.Strings()`. I think it's better to call `sort.Strings()` directly.